osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -712,30 +722,32 @@ BOOL setEnvVars() {
         return FALSE;
     }
     
-    
     // Path of the KALITE_PYTHON environment variable
```

Comment:
I mean we should check if `KALITE_PYTHON` is valid when the app starts, then prompt the user.  

Making the check here in `setEnvVars()`, and returning `FALSE` if it's invalid, doesn't make sense.